signature-aggregator/main/main.go

```diff
@@ -0,0 +1,119 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/ava-labs/avalanchego/message"
+	"github.com/ava-labs/avalanchego/utils/constants"
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"github.com/ava-labs/awm-relayer/peers"
+	"github.com/ava-labs/awm-relayer/signature-aggregator/aggregator"
+	"github.com/ava-labs/awm-relayer/signature-aggregator/api"
+	"github.com/ava-labs/awm-relayer/signature-aggregator/config"
+	"github.com/prometheus/client_golang/prometheus"
+	"go.uber.org/zap"
+)
+
+var version = "v0.0.0-dev"
+
+func main() {
+	fs := config.BuildFlagSet()
+	if err := fs.Parse(os.Args[1:]); err != nil {
+		fmt.Println("Failed to parse flags: %w", err)
+		os.Exit(1)
```

Comment:
Is there a reason this is `os.Exit(1)` instead of `panic`?

Reply:
We might want to `DisplayUsageText()` in this case, and also in any other panic case here that could be helped with the usage text.